src/quacc/recipes/mrcc/core.py

```diff
@@ -0,0 +1,67 @@
+"""Core recipes for MRCC."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from quacc import job
+from quacc.recipes.mrcc._base import run_and_summarize
+
+if TYPE_CHECKING:
+    from ase.atoms import Atoms
+
+    from quacc.types import Filenames, RunSchema, SourceDirectory
+
+
+@job
+def static_job(
+    atoms: Atoms,
+    charge: int = 0,
+    spin_multiplicity: int = 1,
+    method: str = "pbe",
+    basis: str = "sto-3g",
+    mrccinput: dict[str, str] | None = None,
+    mrccblocks: str | None = None,
+    copy_files: SourceDirectory | dict[SourceDirectory, Filenames] | None = None,
+) -> RunSchema:
+    """
+    Carry out a single-point calculation.
+
+    Parameters
+    ----------
+    atoms
+        Atoms object
+    charge
+        Charge of the system.
+    spin_multiplicity
+        Multiplicity of the system.
+    method
+        The method [e.g., PBE or CCSD(T)] to use, this is the value for the calc keyword.
+    basis
+        Basis set
+    mrccinput
+        Dictionary of `mrccinput` swaps for the calculator. To remove entries
+        from the defaults, put a `#` in front of the name. Refer to the
+        [ase.calculators.mrcc.mrcc.MRCC][] calculator for details on `mrccinput`.
```

Comment:
`ase.calculators.mrcc.mrcc.MRCC` doesn't exist, hence why you're getting the docs failure in CI. You can update it to the quacc path for now.